homeassistant/components/mqtt/strings.json

```diff
@@ -19,6 +19,10 @@
     "deprecated_climate_aux_property": {
       "title": "MQTT entities with auxiliary heat support found",
       "description": "Entity `{entity_id}` has auxiliary heat support enabled, which has been deprecated for MQTT climate devices. Please adjust your configuration and remove deprecated config options from your configuration and restart Home Assistant to fix this issue."
+    },
+    "invalid_platform_config": {
+      "title": "Invalid configured MQTT {domain} item",
```

Comment:
The grammar isn't correct here. It should be "Invalidly". Maybe rewrite it to something else though?

Reply:
See https://github.com/home-assistant/core/pull/102496